laura/portfolio/src/main/webapp/style.css

```diff
@@ -9,37 +9,45 @@
 }
 
 body {
-  background-color: ghostwhite;
+  background-color: #2F3131;
```

Comment:
add a comment on what this vlaue is... also you use it multiple times, extracting it to a const might helps a potential refactoring later.